package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "leaderboard-project",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "webpack serve --mode development",
```

Comment:
- [ ] For some reason the app is not opening on the browser after it compiles successfully in the dev-server. Please make sure it runs properly before you submit it for another review. :pray: 

![Screenshot from 2021-11-09 22-48-37](https://user-images.githubusercontent.com/64894927/140994568-dfc21c6c-8c8b-4cf1-aa39-bba0996a00a7.png)